extra/test_release_tool.py

```diff
@@ -76,13 +87,9 @@ def run_main_assert_result(capsys, args, expect=None):
     return captured
 
 
-def test_version_of(capsys):
-    with open(os.path.join(INTEGRATION_DIR, "docker-compose.yml")) as fd:
-        content = fd.read()
-        assert "mendersoftware/gui:" in content
-        if re.search("mendersoftware/gui:.*master", content) is None:
-            # Skip this test for non-master branches.
-            pytest.skip("This test requires master tags in the docker-compose files.")
+def test_version_of(capsys, is_master):
+    if not is_master:
```

Comment:
Suggestion: How about https://docs.pytest.org/en/6.2.x/skipping.html#id1

Reply:
(related to your comment above), I went for this approach to be able to use a `session` coped fixture. I think is safest (and slightly more optimal) to read the file only once and at before any test executes.

The safety point is that this test suite is about reading and writing these components-maps.yml files. If we make a mistake and a test does not clean properly, it could happen that `mendersoftware/gui` gets some other version and then the check will misbehave.

Reply:
In case it wasn't obvious: `skipif` cannot execute a `fixture`.

Reply:
Makes sense!

Reply:
> In case it wasn't obvious: skipif cannot execute a fixture.

TIL